### PR TITLE
Update Mockito to 2.x

### PR DESCRIPTION
**Pull request by slachiewicz**
Mockito 2.7.19
slf4j 1.7.25
logback-classic 1.2.2

### CODE REVIEW

**Comment by khmarbaise** [comment]
Please mention the JIRA issue in your commit message like this:
```
[MNG-6196] ...
```
otherwise it's hard to follow changes in history...also updating the JIRA issue title would be a good idea if you have a good idea what is more descriptive...;-)

**Comment by ebourg** [comment]
#118 also upgrades SLF4J but goes a bit further by removing the patching mechanism.

**Comment by slachiewicz** [comment]
ok, MNG-6196 with slf4j was merged to master - i'm clossing this issue. Mockito will be in diffrent pull.

**Comment by hboutemy** [comment]
great, thank you

**Comment by jira-importer** [comment]
Resolve #7250
